src/pages/steps/AdditionalInformationStep/LabelsStep.tsx

```diff
@@ -118,14 +121,24 @@ function LabelsStep({
             </Text>
           }
         />
-        <Inline spacing={2}>
+        <Inline spacing={3} flexWrap="wrap">
           {labels.map((label) => (
             <Badge
               key={label}
-              variant={BadgeVariants.SECONDARY}
+              variant={BadgeVariants.UNSTYLED}
```

Comment:
The question is if you can't just use `Box`, `Inline` or `Stack` here withe the same props? Instead of adding a unstyled variant to badge. Nothing is special about badge I think

Reply:
Yeah valid point 👍

Reply:
@simon-debruijn done